src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -196,11 +196,25 @@ static ucs_status_t uct_rdamcm_cm_ep_server_init(uct_rdmacm_cm_ep_t *cep,
                                                  const uct_ep_params_t *params)
 {
     struct rdma_cm_event *event = (struct rdma_cm_event *)params->conn_request;
+    uct_rdmacm_cm_t *cm         = uct_rdmacm_cm_ep_get_cm(cep);
     struct rdma_conn_param conn_param;
     ucs_status_t status;
 
-    /* TODO: migrate id if cm is different */
-    ucs_assert(event->listen_id->channel == uct_rdmacm_cm_ep_get_cm(cep)->ev_ch);
+    if (event->listen_id->channel != cm->ev_ch) {
+        /* the server will open the ep to the client on a new CM.
```

Comment:
"on a new CM" -> "on a different CM", then can remove the following clarification

Reply:
If it's all the same to you, i prefer to keep the following clarification. (not all users know what the initial cm was used for)

Reply:
up to you about clarification, my main point that other CM is not mandatory to be "new"

Reply:
this i applied - "on a new CM" -> "on a different CM"

Reply:
I don't see it on GH, make sure you did "git push"

Reply:
i didnt submit it yet